certbot/cli.py

```diff
@@ -1200,6 +1200,14 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         " when the user executes \"certbot renew\", regardless of if the certificate"
         " is renewed. This setting does not apply to important TLS configuration"
         " updates.")
+    helpful.add(
```

Comment:
you'll also want a `default=flag_default("autorenew")` for both of these

Reply:
Done.

Reply:
actually, let's remove this option for simplicity, since we don't need to know whether or not it was set on the command line, we just need to know if the value is true or false.